RELEASE_NOTES_1.0.md

```
@@ -0,0 +1,138 @@
+- Knight Rivers character silhouette design
+- Asset generation script for future PNG conversion
+- All assets generated from code (no external image files)
+
+**Assets created:**
+- `icon.svg` (1024x1024) - App icon
+- `splash-icon.svg` (1284x2778) - Splash screen
+- `adaptive-icon.svg` (1024x1024) - Android adaptive icon
+- `favicon.svg` (48x48) - Web favicon
+
+**Note:** SVG files are used temporarily. For production builds, convert to PNG using:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)